Resources/js/router.test.js

```diff
@@ -386,7 +387,7 @@ function testGetRoutes() {
         blog: 'test'
     });
 
-    assertObjectEquals(expected, router.getRoutes());
+    assertObjectEquals(expected.toObject(), router.getRoutes());
```

Comment:
isn't it just the object passed as argument to `new goog.structs.Map` then ? If yes, it may be simpler to remove the goog Map wrapper instead.

Reply:
My reasoning for doing it this way was to minimise the changes to the tests as far as possible.

Your other comments below have also simplified the install/config process to the point that it adds no additional effort and the test suite itself still relies on the Google library.